packages/react-instantsearch/native.js

```diff
@@ -0,0 +1,5 @@
+import InstantSearch from './src/core/InstantSearch';
```

Comment:
Don't know *exactly* how but I would say we should be able to do something "better" in term of DRY (YAY!):

```js
import createInstantSearch from './src/core/createInstantSearch';
import algoliaClient from 'algoliasearch/native';

export const InstantSearch = createInstantSearch({algoliaClient});
```

And reuse all the code around appId/client creation.

What do you think? This is feasible technically